.github/workflows/test.yml

```
@@ -1,25 +1,45 @@
-name: test
+name: Test Matrix
+
 on:
-- pull_request
+  pull_request:
+  push: { branches: [ master ] }
```

Can we do yaml syntax here instead?

---

\*whine\* Do I _have_ to? 🙎‍♀️ 

```suggestion
  push:
    branches:
    - master
```